.github/scripts/get-release.sh

```diff
@@ -2,21 +2,19 @@
 
 if [[ $GITHUB_REF == refs/tags/v*  ]]
 then
-  if [[ $GITHUB_ACTOR != 'dependabot[bot]' ]]
-  then
-
-    if [[ $GITHUB_PRE_RELEASE == true  ]]
-    then
-      echo "PRE_RELEASE"
-    elif [[ $GITHUB_COMMITISH == 'main' ]]
-    then
-      echo "RELEASE"
-    fi
-   else
-     echo "Dependabot has no permission to publish!"
-     exit 1
-   fi
+	if [[ $GITHUB_ACTOR != 'dependabot[bot]' ]]
+	then
+		if [[ $GITHUB_COMMITISH == 'main' ]]
+		then
+		  echo "RELEASE"
+		else
+		  echo "PRE_RELEASE"
+		fi
+	else
+		echo "Dependabot has no permission to publish!"
+		exit 1
+	fi
```

Comment:
@nmerget please even also keep in mind the prereleases from `main`

Reply:
fixed